### PR TITLE
add ocp 4.18 support for service mesh

### PR DESCRIPTION
**Pull request by vi-patel**
add ocp 4.18 support for service mesh

### CODE REVIEW

**Comment by vi-patel** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.18-lp-interop-servicemesh-interop-aws-fips

**Comment by vi-patel** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.18-lp-interop-servicemesh-interop-aws

File: ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-main__servicemesh-ocp4.18-lp-interop.yaml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: servicemesh-interop-aws
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-main__servicemesh-ocp4.18-lp-interop.yaml:38]: Need to set the 4.18 cron value to day/time job would never be run.
```suggestion
  cron: 0 0 11 6 *
```

**Comment by vi-patel** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.18-lp-interop-servicemesh-interop-aws-fips

**Comment by vi-patel** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.18-lp-interop-servicemesh-interop-aws-fips

**Comment by vi-patel** [comment]
Leaving the issue w/fips running on OCP 4.18 to be resolved later.

**Review by calebevans — APPROVED** [review]
/lgtm

**Pull request merged**